mobile-widgets/qml/MapWidget.qml

```diff
@@ -203,6 +203,8 @@ Item {
 	Image {
 		id: toggleImage
 		x: 10; y: x
+		width: 20
+		height: 20
```

Comment:
This one is too small at 20x20 on my screen. The buttons below are fine at the smaller size